src/index.ts

```diff
@@ -1,12 +1,14 @@
 /**
  * Main
  */
+export * from './account';
 export * from './contract';
-export * from './types';
 export * from './provider';
-export * from './account';
 export * from './signer';
 
+export * from './types';
+export * as types from './types';
```

Comment:
I guess the idea here is to have all types under type. And ``export * from './types';`` is for back compatibility reasons.
My suggestion is to mark it 'to be removed' and open an issue for future version X when it will be removed, so we don't forget it.